lib/status-routes.schema.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
How about including also the current version, of the service? Is this information available?

Reply:
No problem, we have the information from the npm runtime or the package.json of the project.

It will be a non required one as the name and it will be provided as a `packageVersion` decoration on fastify.